mloop/learners.py

```diff
@@ -1984,10 +1999,15 @@ def update_bias_function(self):
         self.cost_bias = self.bias_func_cost_factor[self.params_count%self.bias_func_cycle]
         self.uncer_bias = self.bias_func_uncer_factor[self.params_count%self.bias_func_cycle]
 
-    def predict_biased_cost(self,params):
+    def predict_biased_cost_unscaled(self,params):
         '''
-        Predicts the biased cost at the given parameters. The bias function is:
+        Predict the biased cost at the given parameters.
+        
+        The bias function is:
             biased_cost = cost_bias*pred_cost - uncer_bias*pred_uncer
```

Comment:
Oh I should have changed this when I commented on this part already. Let's take this opportunity to add back ticks as well. I think that should make it render more nicely in the API documentation on readthedocs.

```suggestion
            `biased_cost = cost_bias*pred_cost - uncer_bias*pred_uncer`
```